### DISCUSSION TITLE
我使用的是拼音和五笔混合输入方案，求助排序问题

### DISCUSSION THREAD

**Discussion by shawn998** [rime/home#1421] [Q&A]
我用的是五笔和拼音混合输入方案，首选五笔，拼音作为补充使用。
遇到的问题是当输入二个字母时，仅出现1个二字码的五笔，然后就是N页的拼音首字母方案，比如下图，我想至少首屏可以显示三个字母的五笔提示，而非拼音的方案，怎么配置？
![image](https://github.com/rime/home/assets/30171398/4426cf4d-8fa1-43ae-8d25-d67ea79455f9)

**Comment by ednykissyou** [comment]
我要求不高，只想五笔拼音，首码总是五笔即可。
请问如何搞？
